src/i18n/locales/en_US/account.json

```diff
@@ -13,8 +13,9 @@
     "coupon_applied": "Your coupon code has been applied",
     "coupon_discount": "Coupon discount",
     "coupon_not_valid": "Coupon code is not valid",
-    "credit_card": "Credit Card",
-    "credit_card_number": "Credit card number",
+    "credit_card": "Credit card",
+    "credit_card_name": "Credit card name",
+    "credit_card_number": "1234 5678 9012 3456",
```

Comment:
Does this need translation if it's just numbers?

Reply:
yeah, it should be hardcoded I guess.